draft-ietf-rats-architecture.md

```diff
@@ -376,6 +376,76 @@ create Evidence. Examples of environments that can be used as
+As in any other scenario, this hardware component is is the first Attesting Environment.
+It collects a rather concise number of Claims about the Target Environment.
+The Target Environment in this example is the (U)EFI / BIOS / Firmware.
+After the boot sequence started the Target Environment the
+most important feature of layered attestation is conducted:
+The successfully measured [comment: this is why Henk prefers 'measured' above 'collected'
+because it can be used in English text better] environment that is the
+(U)EFI / BIOS / Firmware now becomes the Attesting Environment.
+
+Now, the duties have been transferred and Layered Attestation takes place.
+The initial Attesting Environment relinquishes its duties to the Target Environment.
+It is important to note that the new Attesting Environment cannot alter
+the content about its own measurements. If the Attesting Environment
+would be able to do that, Layered Attestation would become unfeasible.
+
+In this example the duty of being the Attesting Environment is now
+taken over by the (U)EFI / BIOS / Firmware that was the Attested
+Environment before. This transfer of duty is the essential part of
+Layered Attestation. The (U)EFI / BIOS / Firmware now is the Attesting Environment.
+The next Target Environment is, in this example, a bootloader. There are
+potentially multiple kernels to boot, the decision is imposed on the bootloader.
+Only a integer bootloader will make an appropriate decision. Therefore, Claims about
+the integrity of a bootloader are now collected by the freshly appointed Attesting Environment
+that is the (U)EFI / BIOS / Firmware. Collected Claims have to be stored by the current
+Attesting Environment in a similar shielded and secured manner, so that the next Attesting Environment
+is not capable of altering the collection of claims stored.
+
+Continuing with this example, the bootloader is now in charge of collecting Claims
+about the next execution environment. The next execution environment in this example
+is the kernel to be booted up. Analogously, the next transfer of duties in this
+Layered Attestation example occurs: The duty of being an Attesting Environment is
+transferred to a successfully measured kernel [Henk: we might have to define what successful
+means in this example and beyond]. In this sequence, the kernel is now collecting
+additional Claims and is storing them in a secure and shielded manner.
+
+The essence of this example is a cascade of staged boot environments. Each
+environment (after the initial one that is a root-of-trust) has the duty
+of measuring its next environment before it is started. Therefore, creating
+a layered boot sequence and correspondingly enabling Layered Attestation.
+
+Please note: nested components can compose their on Composite Attesters.
+In consequence, the concept of Layered Attestation is realized in each Attester role.
+If there is a Composite Device enabling Composite Attesters, each Attester in this composite
```

Comment:
I'm confused by the conversation about Composite Device and the unknown thing called Composite Attester in the context of a description of a "Layered Device?" or "Layered Attestation". It seems to me the architecture should clearly describe each independently then let implementation dictate if they should be combined. Combining them shouldn't result in new architecture - not unless there is some anomaly lurking that warrants it.

Reply:
This is a weird artifact while we were trying out new terms in this context. Could we go through this text passage together, please? There is no single good answer it seems to me.